routing_model/index.html

```diff
@@ -193,6 +252,28 @@ <h2>Processing a candidate trying to join our section</h2>
   </div>
 
   <h2>Resource proof from a destination's point of view</h2>
+  <p>In the previous diagram, we ensured an incoming candidate would only be processed for resource_proof if we are the best fit we know and we are not currently processing another node.<br>
+  This leads us here: to the resource proof.<br>
+  We will first add this node to our peer_list with the sate ResourceProof and send them a RelocateResponse.<br>
+  This will let them know we are ready to process them.<br>
+  They will then send their CandidateInfo to everyone in our section.<br>
+  On receiving a valid CandidateInfo, I will send them a ResourceProof RPC. This gives them the "problem to solve". As they solve it, they will send me ResourceProofResponses. These will be parts of the proof. On receiving valid parts, I must send a ResourceProofReceipt. Once they finally send me the last valid part, they passed their resource proof and I vote for ParsecOnline (essentially accepting them as a member of my section).<br>
+  At any time during this process, they may timeout, in which case I will decided to reject them and vote for ParsecPurgeCandidate.<br>
+  This process ends once I reach consensus on either accepting or the candidate (ParsecOnline) or refusing them (ParsecPurgeCandidate).<br>
+  It is possible that both reach consensus, in which case the second one will be discarded in the flow above.
+  </p>
```

Comment:
It is possible for a node to exit this process having done nothing more than seen ParsecOnline or ParsecPurgeCandidate.

Reply:
Well, consensus has to be reached, hasn't it?
Are you trying to say that these have ConsensusMode::Single, so a supermajority is not needed, which is why both may reach consensus?

Reply:
no, I'm trying to say that consensus can be reached without any input for that node.